browser/ui/browser_commands.cc

```diff
@@ -254,6 +260,12 @@ void ToggleVerticalTabStripFloatingMode(Browser* browser) {
       !prefs->GetBoolean(brave_tabs::kVerticalTabsFloatingEnabled));
 }
 
+void ToggleVerticalTabStripExpanded(Browser* browser) {
+  auto* prefs = browser->profile()->GetOriginalProfile()->GetPrefs();
```

Comment:
As kVerticalTabsCollapsed is added to `brave_pref_service_incognito_allowlist.cc`, I hope we can remove `GetOriginallProfile()` here.

Reply:
Good call!